rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/export/catalog/ExportToCatalogConfirmWindow.java

```diff
@@ -183,50 +194,50 @@ private void fillBucketList() {
-                                                                               JSONObject lastNodeSourceRevision = nodeSourceRevisions.get(0)
-                                                                                                                                      .isObject();
+                                                                               JSONArray catalogObjectRevisions = JSONParser.parseStrict(response.getText())
+                                                                                                                            .isArray();
+                                                                               JSONObject lastCatalogObjectRevision = catalogObjectRevisions.get(0)
+                                                                                                                                            .isObject();
                                                                                revisionLabel.setContents("(Last Revision: " +
-                                                                                                         lastNodeSourceRevision.get("commit_time")
-                                                                                                                               .isString()
-                                                                                                                               .stringValue() +
+                                                                                                         lastCatalogObjectRevision.get("commit_time")
+                                                                                                                                  .isString()
+                                                                                                                                  .stringValue() +
                                                                                                          " \"" +
-                                                                                                         lastNodeSourceRevision.get("commit_message")
-                                                                                                                               .isString()
-                                                                                                                               .stringValue() +
+                                                                                                         lastCatalogObjectRevision.get("commit_message")
+                                                                                                                                  .isString()
+                                                                                                                                  .stringValue() +
                                                                                                          "\")");
```

Comment:
Following the comment from Gleb. This back-slash has not space in front.

Reply:
Thanks for the head up. But yes, in this case the quotes should stick to the commit message, hence no space.